cmds/core/yes/yes_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestMain(m *testing.M) {
+	testutil.Run(m, main)
+}
+
+func TestYes(t *testing.T) {
+	cmd := testutil.Command(t)
+	fmt.Println(cmd.Path)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	if err := cmd.Start(); err != nil {
+		log.Fatal(err)
+	}
+
+	in := bufio.NewScanner(stdout)
+
+	for i := 0; i < 20; i++ {
+		if in.Scan() {
+			log.Println(in.Text())
+		} else {
+			log.Println("No output")
```

Comment:
Maybe `t.Fatalf` here because if you're not getting output from the yes cmd, you want to log and fail the test right then

Reply:
don't; use the got/want pattern, e.g.
t.Errorf("Reading output: got no input, want input") for example from Go source:
t.Errorf("got %d want %d", got, want)

Since you are iterating here, go ahead and run it for 10 or 20 times, but then you can use t.Errorf for each failure.

Reply:
Fixed it!